src/datasets/summarizations/trend-regression.summarization.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { queryFactory } from './trend-regression.summarization';
+import { TimeSeriesPoint } from '../metas/types';
+
+describe('queryFactory', () => {

```

Comment:
Could we add a couple tests for the summaries generated? Would be nice to use a couple clean/simple datasets to sanity check that the summaries output are correct. (i.e. the current linearly increases dataset should always generate the same summaries).

Reply:
I added two simple test data to verify the created summaries and validity. Let me know your thoughts about it!

Reply:
Added comments on new tests! Thanks for adding them and expanding test coverage 👍